### PR TITLE
Update remotes key points

### LINKED ISSUE #71: Improve remote repositories key points
Currently have:
```
* origin is typically the name of the remote repository used by GitHub.
* Local and remote repositories are not identical, in general.
* Local and remote repositories are not synchronized automatically.
* push and pull commands only affect the branch currently checked out.
```

* Issue with the first point: the name of the repository is unchanged - "origin" is the name of the _remote_, not the repository.
* The second and third points could be combined, e.g. "Local and remote repositories are not identical, in general, as synchronisation must be performed manually".
* Could add a few extra points to summarise the key commands from this section:
  * `git fetch`, followed by `git status`, shows whether there are any changes to synchronize
  * `git pull` brings changes in the upstream branch to the local branch
  * `git push` synchronizes any committed changes in your local branch with the upstream branch

### PR DESCRIPTION
**Pull request by dc2917**
This PR contains a proposed improvement to the key points section of the remote repositories lesson.

* Clarified the meaning of `origin`
* Combined the two points about synchronising local and remote
* Added points to summarise `fetch`, `pull` and `push` commands

Closes #71

### CODE REVIEW

File: _episodes/l2-02-remote_repositories.md [hunk 1]
```diff
@@ -12,9 +12,14 @@ objectives:
 - "Use the push command to send changes on the local branch to the remote one."
 - "Use the pull command to update your local branch with the remote one."
 keypoints:
-- "origin is typically the name of the remote repository used by GitHub."
-- "Local and remote repositories are not identical, in general."
-- "Local and remote repositories are not synchronized automatically."
+- "origin is the default name used by GitHub to refer to a remote repository."
```
Comment (alexdewar) [inline _episodes/l2-02-remote_repositories.md:15]: Technically it's Git's default:

```suggestion
- "origin is the default name used by Git to refer to a remote repository."
```

**Review by alexdewar — APPROVED** [review]
LGTM modulo one small comment.

I'd forgotten we cover `git push` and `git pull` in the intro course... we go over it again in the intermediate course too. Still, probably no harm in giving students a refresher.

File: _episodes/l2-02-remote_repositories.md (hunk 1 above)
Comment (dc2917) [reply _episodes/l2-02-remote_repositories.md:15]: Ah yes, that's right, thanks

**Comment by dc2917** [comment]
> I'd forgotten we cover git push and git pull in the intro course... we go over it again in the intermediate course too. Still, probably no harm in giving students a refresher.

Yeah, definitely good to remind them in the intermediate course. In this bit we inevitably have to mention branches, but don't actually doing any branching, and so have to tell the students "you'll find out about branches in the intermediate course".

**Pull request merged**